ote/src/cljs/ote/views/transit_visualization/change_icons.cljs

```diff
@@ -47,27 +47,27 @@
             trip-stop-time-changes-lower trip-stop-time-changes-upper different-week-date] :as diff}
     with-labels?]
    [:div (stylefy/use-style (style-base/flex-container "row"))
-    [:div {:style {:width "15%"}}
+    [:div {:style {:width "20%"}}
      [icon-l/icon-labeled
       [ote-icons/outline-add-box {:color (if (= 0 added-trips)
                                            style/no-change-color
                                            style/add-color)}]
       [:span (or added-trips (:gtfs/added-trips diff))      ;; :changes and :changes-route* have different namespace
        (when with-labels? " lisättyä vuoroa")]]]
 
-    [:div {:style {:width "15%"}}
+    [:div {:style {:width "20%"}}
      [icon-l/icon-labeled
       [ote-icons/outline-indeterminate-checkbox {:color (if (= 0 removed-trips)
                                                           style/no-change-color
                                                           style/remove-color)}]
-      [:span (or removed-trips (:gtfs/removed-trips diff))
+      [:span (or removed-trips (:gtfs/removed-trips diff) 0)
```

Comment:
If a default `0` is added here, it should be added for `added-trips` as well. Same problem is there.
Though as we know, there's a backend fix for both now in master.